index.js

```diff
@@ -64,7 +64,11 @@ module.exports = function (target, dest, opts) {
             // set the directory where npm install will be run
             opts.cwd = dest;
             // npm should use production by default when install is npm run
-            if (opts.production) {
+            if (opts.noprod) {
```

Comment:
We will handle `noprod` in cordova-cli. All it will do is set `opts.production = false`

Don't need this here